.travis.yml

```diff
@@ -10,16 +10,16 @@ before_install: "cp .travis.settings.xml $HOME/.m2/settings.xml"
 
 after_success:
   - if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
-    docker login -u "$DOCKER_SDCPLATFORM_USERNAME" -p "$DOCKER_SDCPLATFORM_PASSWORD";
+    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
```

Comment:
Glad you renamed this but just make sure you update the travis environmental variables too